.CI/common.groovy

```diff
@@ -43,8 +68,33 @@ def numLogicalCPU() {
 }
 
 void partest(cache=true, extraArgs='') {
+  if isWindows() {
+
+  sh ("""
+     set OMDEV=C:\OMDev
+	 echo on
+	 (
+     echo cd ${WORKSPACE}/testsuite/partest
+     echo time perl ./runtests.pl -j8 -nocolour -with-xml
+     echo CODE=$?
+     echo if test "$CODE" = 0 -o "$CODE" = 7; then
+     echo   cp -f ../runtest.db.* "${env.RUNTESTDB}/"
+     echo fi
+     echo if test "$CODE" = 0 -o "$CODE" = 7; then
+     echo   exit 0
+     echo else
+     echo   exit $CODE
+     echo fi
+     ) > runTestWindows.sh
+
+     set MSYSTEM=MINGW64
+     %OMDEV%\tools\msys\usr\bin\sh --login -i -c "cd ${WORKSPACE} && chmod +x runTestWindows.sh && ./runTestWindows.sh && rm -f ./runTestWindows.sh"
+  """)
+
+  } else {
```

Comment:
Since the entire function is different, you might as well have created a new function partestWindows (unless the idea is to have a job where you take any machine - Windows, Mac, Linux and try to build).

Reply:
true, i thought about it and i said that for the first try to do it like this, but we should have different ones for windows

Reply:
i actually had mingw.groovy before the merge :)

Reply:
It might be ok to have unix.partest() and mingw.partest() for the different files :)